.github/dependabot.yml

```diff
@@ -2,27 +2,39 @@ version: 2
 updates:
   - package-ecosystem: "cargo"
     directory: "/"
-    labels: ["A0-pleasereview", "A4-insubstantial", "E2-forcemacos"]
+    labels: ["A4-insubstantial", "E2-forcemacos"]
     schedule:
       interval: "weekly"
       day: "friday"
```

Comment:
and consider moving it on sat-sun mb to avoid hanging ci on Friday

Reply:
moved to `Sunday` so that we can have a chill Saturday and handle it on Monday